Leanplum-SDK/Classes/Internal/Leanplum.m

```diff
@@ -1858,6 +1864,19 @@ + (void)maybePerformActions:(NSArray *)whenConditions
     }
 }
 
++ (NSNumber *)fetchCountDownForContext:(LPActionContext *)actionContext
+{
+    NSDictionary *messageConfig = [LPVarCache sharedCache].messageDiffs[actionContext.messageId];
+    NSNumber *countdown = messageConfig[@"countdown"];
+    if (actionContext.isPreview) {
+        countdown = @(5.0);
```

Comment:
how about the 5? 

My understanding is that if a message is a preview, we are adding a delay of 5 hours. Is that what we are doing?

Reply:
5 seconds. Yes this logic is copied from LPActionManager listenForLocalNotifications.

Reply:
Can you please elaborate? What comment are you looking for, that its 5 seconds? Cos countdown is by seconds all over our code.